velox/functions/prestosql/DecimalFunctions.cpp

```diff
@@ -364,7 +364,9 @@ void registerDecimalMultiply(const std::string& prefix) {
       exec::SignatureVariable(
           S3::name(),
           fmt::format(
-              "{a_scale} + {b_scale}",
+              "min({a_scale} + {b_scale}, min(38, {a_precision} + {b_precision}))",
```

Comment:
@rui-mo Rui, is this a fix for a bug discovered by the extended Fuzzer? If so, please, submit it as a separate PR.

Reply:
Yes, it is discovered by the decimal fuzzer test. Here `min(38, {a_precision} + {b_precision}` is the result precision, so the result scale should not exceed it. Extracted this change to https://github.com/facebookincubator/velox/pull/9286, thanks.